paket.dependencies

```diff
@@ -1,18 +1,36 @@
 redirects: on
 source https://www.nuget.org/api/v2
+source https://api.nuget.org/v3/index.json
 
-nuget FsCheck
+nuget FSharp.Core content:None
+nuget FSharp.NET.Sdk  content:None
+nuget FSharp.Compiler.Tools
```

Comment:
the `FSharp.Compiler.Tools` is used for something else other than `FSharp.NET.Sdk`?
becuase always use latest version for that is not a good idea, may be unaligned with `FSharp.NET.Sdk`